plugins/node/opentelemetry-instrumentation-redis-4/src/instrumentation.ts

```diff
@@ -332,6 +338,52 @@ export class RedisInstrumentation extends InstrumentationBase<any> {
     };
   }
 
+  private _getPatchedClientConnect() {
+    const plugin = this;
+    return function connectWrapper(original: Function) {
+      return function patchedConnect(this: any): Promise<void> {
+        const options = this.options;
+
+        const attributes = {
+          [SemanticAttributes.DB_SYSTEM]: DbSystemValues.REDIS,
+          [SemanticAttributes.NET_PEER_NAME]: options?.socket?.host,
+          [SemanticAttributes.NET_PEER_PORT]: options?.socket?.port,
+          [SemanticAttributes.DB_CONNECTION_STRING]: options?.url,
+        };
```

Comment:
This code is a duplication of the one in [`_traceClientCommand` function](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/11ab701051cedca051f52707673b28a26e965389/plugins/node/opentelemetry-instrumentation-redis-4/src/instrumentation.ts#L354).

WDYT about extracting it to `utils` (maybe call it `getClientAttributes`) and calling the function from both places?

Reply:
Good point, moved it to `utils`